src/components/tabs/tab.tsx

```diff
@@ -0,0 +1,216 @@
+/*
```

Comment:
The tab itself. It is also a polymorphic component, but the outer element is always an `li` element.